src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -1441,6 +1438,11 @@ override public IEnumerator GetEnumerator()
         }
 
         /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/GetFieldType/*' />
+#if NET6_0_OR_GREATER
```

Comment:
Rather than adding #if everywhere, another technique would be to just to copy the definition of the [DynamicallyAccessedMembers] attribute into the project, doing #if over that for old TFMs. Of course it won't actually do anything there, but it saves you from having to do #if in each and every place where the attribute is used.

Reply:
I personally prefer not making a no-op `DynamicallyAccessedMembersAttribute`. I agree it would make the code on this side more readable, but it becomes unclear what's actually happening. Imho, we shouldn't add no-op attributes.